firefox-ios/Client/Frontend/Home/Homepage Rebuild/TopSites/TopSitesManager.swift

```diff
@@ -7,17 +7,21 @@ import Common
 import Shared
 import Storage
 
-// TODO: FXIOS-10165 - Add full logic + tests for retrieving top sites
+protocol TopSitesManagerInterface {
```

Comment:
nit; I think it's better to have the comments in the protocol rather than on the implementation. i.e. the comment explaining what are "other sites" could be here instead

Reply:
I moved the comment for `recalculateTopSites` as well and add one for `fetchSponsoredSites`.